project/CONTRIBUTING.md

```diff
@@ -87,6 +87,8 @@ protected function bar($foo, string $name, int $limit)
 }
 ```
 
+Please also note that multiline signatures are allowed when the line is longer than 120 characters.
```

Comment:
@greg0ire AFAIK, you are working on a fixer for that. Maybe could you open an issue here to remove this line when the fixer will be merged and integrated?

Reply:
@Soullivaneuh done